crates/iota-sdk-types/src/crypto/intent.rs

```diff
@@ -112,3 +130,7 @@ pub enum IntentAppId {
     Narwhal = 1,
     Consensus = 2,
 }
+
+impl IntentAppId {
+    crate::def_is!(Iota, Narwhal, Consensus);
```

Comment:
This is outdated, we don't have anything with Narwhal anymore
https://github.com/iotaledger/iota/blob/11060fbcfe23d82b8137fb391880949c51646f64/crates/shared-crypto/src/intent.rs#L37-L40

```suggestion
    Consensus = 1,
}

impl IntentAppId {
    crate::def_is!(Iota, Consensus);
```